migrations/versions/b8dea71fabf2_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
```

Comment:
This entire file has spacing issues and won't actually work when I try to use it.